site/gen-api-docs.sh

```diff
@@ -58,7 +58,6 @@ printf "\n" >> $TITLE
 doc_version=$(grep 'feature publishVersion=' $FILE )
 echo $doc_version
 publish='{{% feature publishVersion="'$VERSION'" %}}'
-expiry='{{% feature expiryVersion="'$VERSION'" %}}'
```

Comment:
I don't understand why this file was modified. This variable is used on line 80 below.

Reply:
Reverted the change in site/gen-api-docs.sh

Reply:
Added the Changelog.md file